extensions/BugModal/template/en/default/bug_modal/common_header.html.tmpl

```diff
@@ -0,0 +1,84 @@
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+[%
+  PROCESS global/variables.none.tmpl;
+
+  title = title
+
+  generate_api_token = 1;
+
+  # these aren't always defined
+  UNLESS bodyclasses.defined;
+    bodyclasses = [];
+  END;
+  UNLESS javascript_urls.defined;
+    javascript_urls = [];
+  END;
+  UNLESS style_urls.defined;
+    style_urls = [];
+  END;
+  UNLESS jquery.defined;
+    jquery = [];
+  END;
+
+  # right now we need yui for tracking flags and needinfo extensions
+  no_yui = 0;
+
+  # add body classes for sec-groups, etc
+  bodyclasses.push("bug_modal");
+
+  # assets
+  javascript_urls.push(
+    "extensions/ProdCompSearch/web/js/prod_comp_search.js",
+    "extensions/BugModal/web/common_bug_modal.js",
+    "extensions/BugModal/web/comments.js",
+    "extensions/BugModal/web/dropdown.js",
+    "js/bugzilla-readable-status-min.js",
+    "js/field.js",
+    "js/comments.js",
+    "js/util.js"
+  );
+  jquery.push(
+    "datetimepicker",
+    "contextMenu",
+    "visibility"
+  );
+  style_urls.push(
+    "extensions/BugModal/web/common_bug_modal.css",
+    "extensions/BugModal/web/dropdown.css",
+    "skins/custom/bug_groups.css",
+    "js/jquery/plugins/datetimepicker/datetimepicker.css",
+    "js/jquery/plugins/contextMenu/contextMenu.css"
+  );
+%]
+
+[% javascript = BLOCK %]
+
+  [%# expose useful data to js %]
+
```

Comment:
I know this how it currently works, but it would be interesting to see if we can add this to the (template toolkit) varaible js_BUGZILLA, which gets serialized automatically.
you'll need to grep the code to see if that's possible, and this is by no means a requirement.